core/client/fs/src/main/java/alluxio/client/file/cache/DefaultMetaStore.java

```diff
@@ -13,29 +13,39 @@
 
 import alluxio.exception.PageNotFoundException;
 
-import java.util.HashSet;
-import java.util.Set;
+import java.util.HashMap;
+import java.util.Map;
 
 /**
  * The default implementation of a metadata store for pages stored in cache.
  */
 public class DefaultMetaStore implements MetaStore {
-  private final Set<PageId> mPageMap = new HashSet<>();
+  /** A map from PageId to page size. */
+  private final Map<PageId, Long> mPageMap = new HashMap<>();
```

Comment:
Do we want hashmap or just a page object that has the size?

Reply:
Actually we may need to store more page information later, so having a page info kind of object makes sense, for now we can keep it a long.